properties-files/openshiftv4_on_osp.properties

```diff
@@ -0,0 +1,86 @@
+ORCHESTRATION_USER=root
+ORCHESTRATION_HOST=
+SSHKEY_TOKEN=
+OPENSHIFT_CLEANUP=false
+OPENSHIFT_INSTALL=true
+OPENSHIFT_POST_INSTALL=true
+OPENSHIFT_POST_CONFIG=true
+OPENSHIFT_DEBUG_CONFIG=false
+OPENSHIFT_CLIENT_LOCATION=
+SCALE_CI_BUILD_TRIGGER=true
+SCALE_CI_BUILD_TRIGGER_URL=
+OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=
+OPENSHIFT_INSTALL_BINARY_URL=
+ENABLE_DITTYBOPPER=true
+CERBERUS_ENABLE=false
+KUBECONFIG_PATH=/root/.kube/config
+CERBERUS_CONFIG_PATH=/root/cerberus.yml
+CERBERUS_IMAGE=quay.io/openshift-scale/cerberus:latest
+CERBERUS_URL=http://0.0.0.0:8080
+WATCH_NODES=true
+WATCH_CLUSTER_OPERATORS=true
+WATCH_NAMESPACES=[openshift-etcd, openshift-apiserver, openshift-kube-apiserver, openshift-monitoring, openshift-kube-controller, openshift-machine-api, openshift-kube-scheduler, openshift-ingress, openshift-sdn]
+CERBERUS_PUBLISH_STATUS=true
+INSPECT_COMPONENTS=false
+SLACK_INTEGRATION=false
+SLACK_API_TOKEN=
+SLACK_CHANNEL=
+COP_SLACK_ID={Monday: , Tuesday: , Wednesday: , Thursday: , Friday: , Saturday: , Sunday: }
+SLACK_TEAM_ALIAS=
+ITERATIONS=5
+SLEEP_TIME=30
+DAEMON_MODE=true
+ENABLE_REMOTE_WRITE=false
+SINCGARS_REMOTE_WRITE_URL=
+OPENSHIFT_INSTALL_APIVERSION=v1
+OPENSHIFT_INSTALL_SSH_PUB_KEY_FILE=/root/.ssh/id_rsa.pub
+OPENSHIFT_INSTALL_PULL_SECRET=
+OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_IMAGE_REGISTRY=
+OPENSHIFT_INSTALL_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE=false
+OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE_VERSION=
+GOPATH=/root/.go
+SCALE_LAB_CLOUD_NAME=
+ANSIBLE_SSH_PASSWORD=
+OPENSTACK_VERSION=
+OSP_EXTERNAL_GATEWAY=
+OSP_EXTERNAL_NET_CIDR=
+OSP_EXTERNAL_ALLOCATION_POOLS_START=
+OSP_EXTERNAL_ALLOCATION_POOLS_END=
+OSP_EXTERNAL_INTERFACE_DEFAULT_ROUTE=
+OPENSHIFT_BASE_DOMAIN=
+OPENSHIFT_CLUSTER_NAME=
+OPENSHIFT_MASTER_COUNT=3
+OPENSHIFT_WORKER_COUNT=5
+OPENSHIFT_MASTER_INSTANCE_TYPE=m5.xlarge
```

Comment:
We might want to change the instance types and other resources as these are specific to AWS.

Reply:
yes, i will change all the default values